src/decoder/tracking_player.cc

```diff
@@ -4,8 +4,8 @@
 using namespace std;
 
 template<class FrameType>
-SerializedFrame TrackingPlayer::decode_and_serialize( const FrameType & frame, const Decoder & source,
-                                                      const Chunk & compressed_frame )
```

Comment:
This is the tabbing style we use everywhere in the code base. Why only change it here?